test/controllers/notes_controller_test.rb

```diff
@@ -876,6 +876,45 @@ def test_search_success
     end
   end
 
+  def test_search_by_user_success
```

Comment:
I see the unit test for display name, but is there also a unit test for filter by id?

Reply:
I've added some more tests with the last commit